docs/reference/streamfield/blocks.md

```diff
@@ -2,7 +2,7 @@
 
 # StreamField block reference
 
-This document details the block types provided by Wagtail for use in [StreamField](streamfield), and how they can be combined into new block types.
+This document details the block types provided by Wagtail for use in [StreamField](streamfield_topic), and how they can be combined into new block types.
```

Comment:
This previously links to https://docs.wagtail.org/en/stable/advanced_topics/testing.html#wagtail.test.utils.form_data.streamfield which doesn't make sense in this context. You can see this is the case in [the current docs](https://docs.wagtail.org/en/stable/reference/streamfield/blocks.html).